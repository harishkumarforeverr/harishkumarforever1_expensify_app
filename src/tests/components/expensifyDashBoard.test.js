import React from "react";
import {shallow} from "enzyme";
import ExpensifyDashBoard from "../../components/expensifyDashBoard";

test ("test/componenet/ExpensifyDashBoard should render it coreectly ",()=>{
      const wrapper=shallow(<ExpensifyDashBoard />);
      expect(wrapper).toMatchSnapshot();
})