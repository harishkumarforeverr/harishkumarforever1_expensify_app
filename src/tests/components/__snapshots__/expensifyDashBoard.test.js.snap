// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test/componenet/ExpensifyDashBoard should render it coreectly  1`] = `
<div
  className="marginComponent"
>
  <Connect(ExpensesSummary) />
  <Connect(ExpensesListFilters) />
  <Connect(ExpensesList) />
</div>
`;
